src/setup_subset_benchmarking.R

```diff
@@ -28,5 +28,6 @@ write_eval_config(
   # score for all locations one forecast date
   overwrite_benchmark = TRUE, # Set as TRUE if want to save outputs of
   # benchmarking in directory,
-  name_of_config = "benchmark_config"
+  name_of_config = "benchmark_config",
+  wwinference_version = "v0.1.0"
```

Comment:
Use the hash here?

Reply:
I made this so that its an option to either specify or use the hash-- these results were run using the v0.1.0 (even though that is not currently what is installed locally) and so I want to just specify manually here.